pages/coq-team.html

```diff
@@ -0,0 +1,486 @@
+  margin-bottom: 15px;
+  margin-top: 0px;
+}
+
+.name {
+    display: inline;
+    text-align: center;
+    font-weight: bold;
+    font-size: 18pt;
+    margin-top: 20px;
+    margin-bottom: 5px;
+    /* color: red;*/
+  } 
+.nickname {
+    /* display: inline; */
+    text-align: center;
+    font-style: bold;
+    font-family: 'Courier New', Courier, monospace;;
+    font-size: 10pt;
+    margin-top: 3px;
+    margin-bottom: 3px;
+    /* color: red;*/
+}
+.position {
+    text-align: center;
+    font-style: italic;
+    font-size: 10pt;
+    margin-top: 3px;
+    margin-bottom: 3px;
+    /* color: red;*/
+} 
+.role {
+  text-align: center;
+  font-weight: bold;
+  font-size: 12pt;
+  margin-top: 3px;
+  margin-bottom: 3px;
+  /* color: red;*/
+} 
+.components {
+    text-align: center;
+    font-style: normal;
+    font-size: 8pt;
+    font-weight: lighter;
+    margin-top: 5px;
+    margin-bottom: 3px;
+    /* color: red;*/
+  }
+
+
+  </style>
+  <script type="text/javascript" language="javascript">
+      function getElementsByClass (className) {
+        var all = document.all ? document.all :
+          document.getElementsByTagName('*');
+        var elements = new Array();
+        for (var i = 0; i < all.length; i++)
+          if (all[i].className == className)
+            elements[elements.length] = all[i];
+        return elements;
+      }
+    
+      function makeLinks(className, url, prefix) {
+        nodes = getElementsByClass(className);
+        for(var i = 0; i < nodes.length; i++) {
+          node = nodes[i];
+          text = node.innerHTML
+          node.innerHTML = '<a href="' + url + text + '">' + prefix + text + '</a>';
+        }
+      }
+  </script>
+</#def>
+
+<#def CONTENT>1</#def>
+<#include "incl/header.html">
+
+<div class="coordinator">
+  <div class="title">Project Coordinator</div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/mattam82.png" alt="Matthieu Sozeau" width="170px"/></div>
+    <div class="name"><a href="https://mattam.org">Matthieu Sozeau</a></div><div class="nickname">mattam82</div>
+    <div class="position">Inria Researcher in the Gallinette Team, Nantes, France</div>
+    <div class="role">RM: 8.5</div>
+    <div class="components">General Maintenance, Theory, Kernel, Elaboration, Unification, Typeclasses, Program, Universes, Equations, MetaCoq</div>
+  </div>
+</div>
+<div class="team">
+  <div class="team-name"><h1>Core Team</h1></div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/YvesBertot.png" alt="Yves Bertot" width="200px"/></div>
+    <div class="name"><a href="http://www-sop.inria.fr/members/Yves.Bertot/research.html">Yves Bertot</a></div><div class="nickname">YvesBertot</div>
+    <div class="position">Inria Researcher, Head of the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="role">Head of the Coq Consortium</div>
+    <div class="components">Coq'Art co-author, Function, Coq-Tutorial, Documentation, 
```

Comment:
Did you really mean `coq-tutorial` here or `plugin-tutorial`?

Reply:
You're right, of course